src/Exporter/Plugin/ShippingMethodResourcePlugin.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+namespace FriendsOfSylius\SyliusImportExportPlugin\Exporter\Plugin;
+
+use Sylius\Component\Core\Model\ChannelInterface;
+use Sylius\Component\Core\Model\ShippingMethodInterface;
+use Sylius\Component\Shipping\Model\ShippingMethodTranslationInterface;
+
+final class ShippingMethodResourcePlugin extends ResourcePlugin
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function init(array $idsToExport): void
+    {
+        parent::init($idsToExport);
+
+        /** @var ShippingMethodInterface $resource */
+        foreach ($this->resources as $resource) {
+            if (null !== $resource->getZone()) {
+                $this->addDataForResource($resource, 'Zone', $resource->getZone()->getCode());
+            }
+
+            if (null !== $resource->getCategory()) {
+                $this->addDataForResource($resource, 'Category', $resource->getCategory()->getCode());
+            }
+
+            $channels = [];
+            /** @var ChannelInterface $channel */
+            foreach ($resource->getChannels() as $channel) {
+                $channels[] = $channel->getCode();
+            }
+
+            $this->addDataForResource($resource, 'Channels', $channels);
```

Comment:
Since we removed channels from the import, we can remove them from the export as well.